sqlmesh/core/dialect.py

```diff
@@ -429,6 +429,10 @@ def _parse_props(self: Parser) -> t.Optional[exp.Expression]:
             lambda: _parse_macro_or_clause(self, self._parse_when_matched),
             optional=True,
         )
+    elif name == "incremental_predicates":
+        value = self._parse_wrapped(
+            lambda: _parse_macro_or_clause(self, self._parse_conjunction), optional=True
+        )
```

Comment:
If this is only applicable to `MERGE`, should we rename it appropriately? Current naming feels too vague to me.

Reply:
I was considering this as well, for now I thought since dbt uses the same [incremental_predicates](https://docs.getdbt.com/docs/build/incremental-strategy#about-incremental_predicates) name I think it will be easier for people that migrate off dbt to immediately know where it is used, but can rename it to something else

Reply:
As a follow up to our internal discussion revised the name to `merge_filters`